src/libraries/System.Text.Json/gen/Model/ClassType.cs

```diff
@@ -1,13 +1,9 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace System.Text.Json.SourceGeneration
 {
-    internal enum ClassType
+    public enum ClassType
```

Comment:
nit: we can remove the ctors part of line 9. Also this is tangential but at this point are there any types that we don't emit source for? I remember multi-dim arrays but we might have switched to generating metadata that throws `NotSupportedException` in some way.